lib/util.c

```diff
@@ -73,9 +73,7 @@ json_object* generic_reply(json_object* msg)
     json_object_object_add(reply_body, "in_reply_to", reply_in_reply_to);
 
     // Generate reply type: "XXX_ok"
-    json_object* type =
-        json_object_object_get(json_object_object_get(msg, "body"), "type");
-    const char* name = json_object_get_string(type);
+    const char* name = msg_type(msg);
```

Comment:
😆  I didn't even notice you'd implemented this convenience function already!

Reply:
np! I think we might've introduced the convenience function after this code was written :)